### DISCUSSION TITLE
Time Steps & Time of Departure Customization

### DISCUSSION THREAD

**Discussion by filofossati** [iiasa/message_ix#610] [MESSAGE modeling] [answered]
Hi There! I have two questions:

if it is possible to reduce the time steps simulation from 5 years down to 1?
And if it is possible to manually change the time of departure of the simulation? for instance right now I believe the base year is 2020; but I was wondering if I could change that and for example making the simulation begin in 2019?
Thanks a lot in advance!
Filo

**Comment by khaeru** [comment, marked as answer]
Hi @filofossati —welcome, and thanks for your questions! I have moved these to the Discussions forum, because they are not related to any bug or desired addition to the code, only understanding existing functionality:

> if it is possible to reduce the time steps simulation from 5 years down to 1?

Yes. Please read the dedicated documentation page [“Years, periods, and time slices”](https://docs.messageix.org/en/stable/time.html), in particular the description of the `duration_period` parameter.

> And if it is possible to manually change the time of departure of the simulation? for instance right now I believe the base year is 2020; but I was wondering if I could change that and for example making the simulation begin in 2019?

Yes. The concepts here are explained in the same place. The function [`add_horizon()`](https://docs.messageix.org/en/stable/api.html#message_ix.Scenario.add_horizon) is provided to simultaneously configure several sets and parameters related to time.

If this answer covers your initial questions, please mark it ✅ . If you have further questions and cannot answer them by reading the documentation, please ask a follow-up here or in a new discussion.

**Comment by filofossati** [comment]
Thanks for your answers @khaeru!I will dig into it and let you know if I have other questions!
